src/main/java/uk/gov/hmcts/ccd/data/definition/CachedCaseDefinitionRepository.java

```diff
@@ -51,29 +49,23 @@ public CachedCaseDefinitionRepository(@Qualifier(DefaultCaseDefinitionRepository
     @Override
     public List<CaseTypeDefinition> getCaseTypesForJurisdiction(final String jurisdictionId) {
         return caseTypesForJurisdictions.computeIfAbsent(jurisdictionId,
-                                                         caseDefinitionRepository::getCaseTypesForJurisdiction);
+            caseDefinitionRepository::getCaseTypesForJurisdiction);
     }
 
     @Override
     public CaseTypeDefinition getCaseType(final String caseTypeId) {
         CaseTypeDefinitionVersion latestVersion = this.getLatestVersion(caseTypeId);
-        return this.getCaseType(latestVersion.getVersion(), caseTypeId);
+        return caseDefinitionRepository.getCaseType(latestVersion.getVersion(), caseTypeId);
+    }
+
+    @Override
+    public CaseTypeDefinition getCaseType(int version, String caseTypeId) {
+        return caseDefinitionRepository.getCaseType(version, caseTypeId);
     }
 
     @Override
-    public CaseTypeDefinition getCaseType(final int version, final String caseTypeId) {
-        final int fromHour = applicationParams.getRequestScopeCachedCaseTypesFromHour();
-        final int tillHour = applicationParams.getRequestScopeCachedCaseTypesTillHour();
-        final int currentHour = LocalDateTime.now().getHour();
-        final boolean withinTimeInterval = currentHour >= fromHour  && currentHour < tillHour;
-        final boolean cacheSwitchOnForCaseType = applicationParams.getRequestScopeCachedCaseTypes().stream()
-            .anyMatch(ct -> ct.equalsIgnoreCase(caseTypeId));
-        if (withinTimeInterval && cacheSwitchOnForCaseType) {
-            return caseTypes.computeIfAbsent(format(CASE_TYPE_KEY_FORMAT, caseTypeId, version),
-                e -> caseDefinitionRepository.getCaseType(version, caseTypeId));
-        } else {
-            return caseDefinitionRepository.getCaseType(version, caseTypeId);
-        }
```

Comment:
These changes shouldn't be required for this PR (and weren't in #1755)- we can keep them as is in master 😄

Reply:
DONE :) :)